apps/sched/clkinfo.js

```diff
@@ -4,6 +4,7 @@
   const iconAlarmOff = atob("GBiBAAAAAAAAAAYAYA4AcBx+ODn/nAP/wAf/4A/n8A/n8B/n+B/n+B/nAB/mAB/geB/5/g/5tg/zAwfzhwPzhwHzAwB5tgAB/gAAeA==");
   const iconTimerOn = atob("GBiBAAAAAAAAAAAAAAf/4Af/4AGBgAGBgAGBgAD/AAD/AAB+AAA8AAA8AAB+AADnAADDAAGBgAGBgAGBgAf/4Af/4AAAAAAAAAAAAA==");
   const iconTimerOff = atob("GBiBAAAAAAAAAAAAAAf/4Af/4AGBgAGBgAGBgAD/AAD/AAB+AAA8AAA8AAB+AADkeADB/gGBtgGDAwGDhwfzhwfzAwABtgAB/gAAeA==");
+  const iconEvent = atob("Hh6BAf////////////////////gAAH/AAAD/P//z/P//z/P//z/P//z/AAAD/AAAD/AAAD/AAAD/AAMD/AAcD/AA4D/AxwD/A7gD/AfAD/AOAD/AAAD/AAAD/AAAD/AAAD/gAAH////////////////////w");
```

Comment:
I used the "Transparency to color" option and now it seems to work:

```suggestion
  const iconEvent = atob("Hh4BAAAAAAAAAAAAAAAAAAAAB///gD///wDAAAwDAAAwDAAAwDAAAwD///wD///wD///wD///wD//PwD/+PwD/8fwD84/wD8R/wD+D/wD/H/wD/v/wD///wD///wD///wB///gAAAAAAAAAAAAAAAAAAAAA=");
```

Reply:
It will also respect the theme (tried with setting fg to green and bg to red).

Reply:
I guess black in 1 bit will correspond to bg and white to fg, so that was the problem.

Reply:
Another thing, the icon I add with this suggestion is larger in pixels than e.g. `iconTimerOn`.

Reply:
Aha nice! One thing I meant to ask - if you're happy with that icon I can try splicing in the on/off part so it matches up with the display of alarms. But perhaps in another PR, like the weekday-letter one

Reply:
That would look nice (also in the alarms app), good idea! But I agree - lets do it later :)